procedures/foreman/testweek.md.erb

```diff
@@ -0,0 +1,100 @@
+Foreman <%= full_version %> has been announced. **We need your help!** It's easy to get your hands dirty and help us with testing:
+
+1. Install Foreman RC version (the most up to date one)
+1. Pick a scenario from this post or add your own scenario
+1. If you find an issue, file it in the RedMine and **make a comment in this thread** linking the issue
+1. Mark the scenario as checked in this OP (this is an editable wiki post), here is syntax of checked and unchecked lines (you can click on checkboxes directly with mouse):
+   - [ ] Unchecked
+   - [x] Checked
+
+**Installation**
+* Install Foreman using existing script/forklift/beaker
+  * [ ] Forklift
+  * [ ] My own install script
+* Install Foreman *manually* by following our [installation guide](https://theforeman.org/manuals/latest/index.html)
+  * [ ] RHEL / CentOS stable
+  * [ ] Debian / Ubuntu stable
+* Install Foreman *manually* by following our new [installation guide](https://docs.theforeman.org)
+  * [ ] RHEL / CentOS
+  * [ ] Debian / Ubuntu
```

Comment:
For installation and upgrades should we explicitly add versions of Operating systems? I think this will help users to have awareness about what is supported platform and where testing should happen.
To add this maybe procedure should accept OS versions?

Reply:
I would think that's OK. Explicit checkboxes are OK.